### PR TITLE
Improve audio scanner robustness

### LINKED ISSUE #726: Scanner hangs with AVI file
Hi,

If you run **mopidy local scan** command and you have any AVI file in your media directory, the process will hang with no error message.

Therefore, it would be interesting to include .avi/.AVI files in default exclude section (mopidy.conf).

Thanks!

### PR DESCRIPTION
**Pull request by adamcik**
Likely fixes #726. Sadly my plan for fixing #797 falls a bit short, as only having the scanner handle it won't do much good.

### CODE REVIEW

**Comment by adamcik** [comment]
Forgot the changelog... I'll get to that tomorrow.

File: mopidy/audio/scan.py [hunk 1]
```diff
@@ -160,3 +171,28 @@ def _process(pipeline, timeout_ms):
...
+    gobject.threads_init()
+
+    scanner = Scanner(5000)
+    for uri in sys.argv[1:]:
+        if not gst.uri_is_valid(uri):
+            uri = path.path_to_uri(os.path.abspath(uri))
+        try:
+            result = scanner.scan(uri)
+            for key in ('uri', 'mime', 'duration', 'playable', 'seekable'):
+                print '%-20s   %s' % (key, getattr(result, key))
```
Comment (jodal) [inline mopidy/audio/scan.py]: Should use `from __future__ import print_function` and `print()` for Py3 compat.

**Comment by jodal** [comment]
Rest looks good :-)

**Comment by adamcik** [comment]
Fixed.

**Pull request merged**